azurerm/resource_arm_automation_job_schedule_test.go

```diff
@@ -0,0 +1,308 @@
+package azurerm
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMAutomationJobSchedule_basic(t *testing.T) {
+	resourceName := "azurerm_automation_job_schedule.test"
+	ri := rand.Int()
```

Comment:
We have a function for this already: 
```suggestion
	ri := tf.AccRandTimeInt()
```